src/components/shared/keyline.tsx

```diff
@@ -6,8 +6,8 @@ import { from } from '@guardian/src-foundations/mq';
 import { Design } from 'item';
 
 const BaseStyles = css`
-    height: 12px;
-    margin-bottom: 4px;
```

Comment:
I think this is useful underneath liveblog keylines

Reply:
The `margin-bottom`?

Reply:
I guess this can be replaced by Simon's work in the design system!